integration-tests/known_networks.go

```diff
@@ -27,6 +33,38 @@ var (
 	// generalEVM is a customizable network through environment variables
 	generalEVM *blockchain.EVMNetwork = blockchain.LoadNetworkFromEnvironment()
 
+	NetworkAlpha = &blockchain.EVMNetwork{
```

Comment:
Not a huge fan of the name. Maybe something more explicit like `SimulatedSourceCCIP`?